modules/wkt/test/wkb/encode-wkb.spec.ts

```diff
@@ -0,0 +1,143 @@
+import test from 'tape-promise/tape';
```

Comment:
Do we need any attribution?

Reply:
The test data is the same as what's used for for `WKBLoader`, an attribution sentence is here: https://github.com/visgl/loaders.gl/blob/master/modules/wkt/test/data/README.md